.gitlab-ci.yml

```diff
@@ -97,11 +97,11 @@ build-review-image:
   needs: []
   interruptible: true
   variables:
-    BRANCH_TAGGING_STRING: ""
+    BRANCH_TAGGING_STRING: ''
```

Comment:
There's a few line changes like this, are they needed?

Reply:
> There's a few line changes like this, are they needed?

Not strictly necessary, they're from my Prettier auto-formatter.

We apply Prettier already to a good chunk of the YAML files in this repository and I'd feel semi-strongly that it should be applied / enforced here as well, but absent the tooling for that, I think the occasional manual "sync-up" is okay?